edly-io/edx-platform
# Add dummy migration for migration rollback testing

**Jira Ticket:** [EDLY-1708](https://edlyio.atlassian.net/browse/EDLY-1708)
**Related Ticket:** [EDLY-911](https://edlyio.atlassian.net/browse/EDLY-911)

This branch was create to create a dummy migration which can be used by the devops to test release rollback including the migrations.

**Testing Instructions:**

1. Create a new dummy release tag for testing this change on stage multisites `edly-test-1.1.0`
2. Devops will deploy this release tag
3. Verify that changes are on stage  (the `dummy_field` in `EdlyUserProfile` model)
4. Devops will rollback this release by reverting to previous stable release tag `edly-2.3.0` (https://github.com/edly-io/edx-platform/releases/tag/edly-2.3.0)
5. Verify that ansible playbook for rollback is successfully commpleted without any error
6. Verify that the changes are stage (the `dummy_field` in `EdlyUserProfile` model) were reverted 


_**Note:** Don't merge this PR. Its created for testing purpose only._

[EDLY-1708]: https://edlyio.atlassian.net/browse/EDLY-1708?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
[EDLY-911]: https://edlyio.atlassian.net/browse/EDLY-911?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
